extension/docs/HowToRun.md

```diff
@@ -147,6 +148,7 @@ Other configurations can be set as direct child attributes in `ADYEN_INTEGRATION
 | `logLevel`                   | The log level (`trace`, `debug`, `info`, `warn`, `error`, `fatal`).                                                                                                                | NO       | `info`                                                                                                |
 | `keepAliveTimeout`           | Milliseconds to keep a socket alive after the last response ([Node.js docs](https://nodejs.org/dist/latest/docs/api/http.html#http_server_keepalivetimeout)).                      | NO       | Node.js default (5 seconds)                                                                           |
 | `basicAuth`                  | Boolean attribute to enable/disable basic authentication to prevent unauthorized 3rd-party from accessing extension endpoint                                                       | NO       | false                                                                                                 |
+| `logSensitiveData`           | Boolean attribute. When `true`, all data from Adyen will be saved in the interface interaction and in the custom fields                                                            | NO       | false                                                                                                 |
```

Comment:
`log` prefix might confusing for users, I guess logic is more related to removing additional data, so I would use `removeAdditionalData` and default value as true. 

For description, 

Boolean attribute. When set to `false`, [additionalData](https://docs.adyen.com/api-explorer/#/CheckoutService/v67/post/payments__resParam_additionalData) which contains additional information about the payment returning from the Adyen responses `will be saved` in the interface interaction and in the custom fields.

Reply:
Sounds the same to me :) Anyway, we don't remove only additionalData and the idea of this field is to remove sensitive data, so I change the name to `removeSensitiveData`: https://github.com/commercetools/commercetools-adyen-integration/pull/745/commits/f0e09786af71fb0ac001e8f609f197014b20a4d9